src/status_im/constants.cljs

```diff
@@ -609,3 +609,10 @@
 
 (def ^:const min-token-decimals-to-display 6)
 (def ^:const swap-proposal-refresh-interval-ms 15000)
+
+(def router-error-code-generic "0")
+(def router-error-code-paraswap-custom-error "WPP-030")
+(def router-error-code-price-timeout "WPP-037")
+(def router-error-code-not-enough-liquidity "WPP-038")
+(def router-error-code-price-impact-too-high "WPP-039")
+(def router-error-code-not-enough-native-balance "WR-002")
```

Comment:
More of a code style comment, but can we have a map for the errors to make it easier on the eyes :sweat_smile:? I see that we do it is as above in mostly all cases for constants, but to me it would be a bit cleaner this way:

```clj
(def router-errors
  {:error/generic              "0"
   :error/paraswap-custom      "WPP-030"
   :error/price-timeout        "WP-037"
   :error/not-enough-liquidity "WPP-038"})

;; when asserting the error code
(= error-response-code
   (:error/not-enough-liquidity constants/router-errors))
``` 

it could also be just me, so I wonder if others have any preferences, cause going through the constants namespace gets more and more annoying recently and I think it's due to the structure of things.

cc: @ilmotta @ulisesmac is there some clojure-friendly way of doing this?

Reply:
Yeah, I started using a map and then changed to something similar as the desktop implementation. Open to suggestions on how to improve the approach :)